---
https://bugs.gentoo.org/show_bug.cgi?id=255750
https://sourceforge.net/tracker/index.php?func=detail&aid=2543239&group_id=142&atid=100142

This removes some obsolete calls to dbus-0.33.0 which were introduced in 2005,
according to the internal ChangeLog of the package.


--- a/src/screem-dbus.c	2009-01-18 12:26:29.000000000 +0100
+++ b/src/screem-dbus.c	2009-01-18 21:57:55.000000000 +0100
@@ -129,19 +129,9 @@
 	
 #ifdef HAVE_DBUS
 	if( priv->conn ) {
-#if DBUS_VERSION > 33000
-		dbus_connection_close( priv->conn );
-#else
-		dbus_connection_disconnect( priv->conn );
-#endif
 		dbus_connection_unref( priv->conn );
 	}
 	if( priv->sconn ) {
-#if DBUS_VERSION > 33000
-		dbus_connection_close( priv->sconn );
-#else
-		dbus_connection_disconnect( priv->sconn );
-#endif
 		dbus_connection_unref( priv->sconn );
 	}
 #endif
